apis/sgv2-quarkus-common/CONFIGURATION.md

```diff
@@ -76,7 +76,25 @@
 | `stargate.queries.consistency.schema-changes` | `String` | `LOCAL_QUORUM` | Consistency level to use for C* queries that are performing the schema changes.  |
 | `stargate.queries.consistency.writes`         | `String` | `LOCAL_QUORUM` | Consistency level to use for C* queries that are inserting or updating the data. |
 | `stargate.queries.consistency.reads`          | `String` | `LOCAL_QUORUM` | Consistency level to use for C* queries that are reading the data.               |
-| `stargate.queries.serial-consistency`         | `String` | `SERIAL`      | Serial consistency level to be used for C* queries.                              |
+| `stargate.queries.serial-consistency`         | `String` | `SERIAL`       | Serial consistency level to be used for C* queries.                              |
+
+### Additional Logging Configuration
+*Configuration mapping for the additional logging that are not covered by Quarkus out of the box such as request body, defined by [LoggingConfig.java](src/main/java/io/stargate/sgv2/api/common/config/LoggingConfig.java).*
+
+> NOTE: In order to use these features, you need to set the log level of `io.stargate.sgv2.api.common.logging.LoggingFilter` to `DEBUG` by doing below:
+```shell
+export QUARKUS_LOG_CATEGORY__IO_STARGATE_SGV2_API_COMMON_LOGGING_LOGGINGFILTER__LEVEL=DEBUG
+```
+
+| Property                                            | Type      | Default | Description                                                                                              |
+|-----------------------------------------------------|-----------|---------|----------------------------------------------------------------------------------------------------------|
+| `stargate.api.logging.enabled`                      | `boolean` | `false` | API level request logging is enabled.                                                                    |
+| `stargate.api.logging.enabled-tenants`              | `Set`     | `all`   | If set, only the requests from the tenants listed here will be logged. If empty, all tenants are logged. |
+| `stargate.api.logging.enabled-paths`                | `Set`     | `all`   | If set, only the requests to the paths listed here will be logged. If empty, all paths are logged.       |
```

Comment:
Is "all" then same as empty? (default is "all")

Reply:
Looking at later definitions, it sounds like constant `"all"` means what "empy" refers to -- or "all" and "" are aliases. Maybe better to say "If set to value other than 'all', ..."?

Reply:
I mean all the tenants by 'all' here. So, the 'all' in `enabled-paths` or leaving it not given means, log requests to all paths when `stargate.api.logging.enabled` is set to `true` and then we can control on which path to be logged by setting paths explicitly.

And all these parameter's values are treated as AND condition. 

**For example:**
If below is the configuration, then this means that all the `POST` requests to the path `/api/rest/v2/keyspaces` from the tenant `tenant-id1-1` that fails with `400` error will be logged along with its body.

```properties
stargate.api.logging.enabled-tenants=tenant-id-1
stargate.api.logging.enabled-paths=/api/rest/v2/keyspaces
stargate.api.logging.enabled-error-codes=400
stargate.api.logging.enabled-methods=POST
stargate.api.logging.request-body-logging-enabled=true
```

If the `stargate.api.logging.enabled-tenants=tenant-id-1` is not set, then it will be set to `all` by default, which means that all the `POST` requests to the path `/api/rest/v2/keyspaces` from **_any tenant_** that fails with `400` error will be logged along with its body.

Reply:
I guess I am just wondering why have both "" and "all" and not just, say, "".

I do understand the logic itself, nothing wrong with that (AND is what I'd expect and I think most users too).

Reply:
The `Default` column has what is the default value when it is not provided, which says 'all'. Having empty string "" as the default value will be confusing I believe.

Reply:
@tatu-at-datastax I fixed the config description as well as you suggested.